payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/DropdownFieldUI.kt

```diff
@@ -129,7 +129,6 @@ internal fun DropdownLabel(
     val color = PaymentsTheme.colors.placeholderText
     Text(
         stringResource(label),
-        color = if (enabled) color else color.copy(alpha = ContentAlpha.disabled),
-        style = MaterialTheme.typography.caption
+        color = if (enabled) color else color.copy(alpha = ContentAlpha.disabled)
```

Comment:
That's a good idea. Maybe a getColor and a getColorRespectingAlpha or something. I'll add this to my backlog rather than doing it here because I want to keep this PR focussed on typography.